README.md

```diff
@@ -255,11 +258,31 @@ topology.metrics.consumer.register:
   - class: "com.verisign.storm.metrics.GraphiteMetricsConsumer"
     parallelism.hint: 1
     argument:
+      metrics.reporter.name: "com.verisign.storm.metrics.reporters.GraphiteReporter"
       metrics.graphite.host: "graphite.example.com"
       metrics.graphite.port: "2003"
       metrics.graphite.prefix: "storm.test"
       metrics.graphite.min-connect-attempt-interval-secs: "5"
 ```
+##### Reporting Metrics to Kafka
+
+```yaml
+---
+### Note: This is Storm's storm.yaml configuration file
+### Note: Kafka producer settings can be passed as registration arguments.
+
+# Controls the time interval between metric reports
+topology.builtin.metrics.bucket.size.secs: 10
+topology.metrics.consumer.register:
+  - class: "com.verisign.storm.metrics.GraphiteMetricsConsumer"
+    parallelism.hint: 1
+    argument:
+      metrics.reporter.name: "com.verisign.storm.metrics.reporters.KafkaReporter"
+      metrics.graphite.prefix: "storm.test"
+	  metrics.kafka.topic: "metricsTopic"
+	  metadata.broker.list: "broker1.example.com:9092,broker2.example.com:9092,broker3.example.com:9092"
```

Comment:
I can see how changing the parameter name to, say `metrics.kafka.broker.list` would more clearly indicate that it's required, I'll make the change.